packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -365,12 +371,15 @@ const RestrictedMenuItem = (props: RestrictedMenuItemProps) => {
     );
 
   if (props.featureToggle) {
-    return (
-      <ToggleFeature flag={props.featureToggle}>
-        {permissionsWrapper}
-      </ToggleFeature>
-    );
+    // A regular short-lived feature toggle
+    if (flagVariation === true) return permissionsWrapper;
```

Comment:
So either it's a boolean then we can use it. Could also be another type guard or more sophisticated check but this should do.